content/applications/productivity/knowledge/properties.rst

```diff
@@ -0,0 +1,82 @@
+==========
+Properties
+==========
+
+Properties are fields containing data and that can be added to articles by any user with **write**
+access. These fields are shared between all the child articles and article items that are under the
+same parent.
+
+.. note::
+   To be able to add properties, an article must be either a **child article** or an **article
+   item**.
+
+Add property fields
+===================
+
+Hover above the first-level header to make the buttons appear. Click on
+:menuselection:`⚙ Add Properties --> Field Type`, select the type and add a default value if needed.
+To make the fields appear in **kanban views**, check :guilabel:`View in Kanban` as well. To validate
+and close the property creation window, click anywhere.
+
+.. image:: properties/fields.png
+   :align: center
+   :alt: Dropdown of property fields types
+
+The different types assess what the field content can be:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Types
+     - Uses
+   * - :guilabel:`Text`
+     - Allows adding any content with no restriction.
+   * - :guilabel:`Checkbox`
+     - Add a checkbox.
+   * - :guilabel:`Integer`
+     - Allows adding integer numbers.
+   * - :guilabel:`Decimal`
+     - Allows adding any number.
+   * - :guilabel:`Date`
+     - Allows selecting a date.
+   * - :guilabel:`Date & Time`
+     - Allows selecting a date and time.
+   * - :guilabel:`Selection`
+     - | Add a drop-down selection menu with restricted values that have been set at the property
+         creation.
+       | To set it up, click :menuselection:`Add a property --> Field Type --> Selection -->
+         Add values --> Enter predetermined values --> Press enter to validate --> repeat --> Click
+         anywhere to close the property creation window`. You can enter as many values as needed.
+   * - :guilabel:`Tags`
+     - | Allows creating as many tags as needed. You can choose from already created tags, or create
```

Comment:
Allows creating as many tags as needed. You can choose from existing tags or create

Reply:
Changed it :)